src/swaps/thorchain/ThorchainSwapProvider.ts

```diff
@@ -504,7 +504,10 @@ class ThorchainSwapProvider extends SwapProvider {
     return null;
   }
 
-  async getMin(_quoteRequest: QuoteRequest) {
+  async getMin(quote: QuoteRequest) {
+    if (quote.from === 'BTC') { // Thorchain swaps have to be more than 10000 sats
+      return new BN(0.00010001)
```

Comment:
Do you believe it's a good idea to have a mapping that says:

[BTC]: BN(0.00010001)

so we can easily extend later if needed?


the usage would be

`return map[quote.from] || BN(0)`

Reply:
sure